lfc/core/src/main/kotlin/org/lflang/generator/uc/UcNetworkChannel.kt

```diff
@@ -242,17 +242,24 @@ class UcTcpIpChannel(
 class UcCoapUdpIpChannel(
```

Comment:
Is this homogeneous enough that one code-generator class works potentially for all platforms?

Reply:
We will find out when we add support for native and Zephyr